tests/FakeItEasy.IntegrationTests/StringArgumentConstraints.cs

```diff
@@ -1,19 +1,17 @@
 namespace FakeItEasy.IntegrationTests
 {
     using System;
-    using System.Globalization;
 
     public static class StringArgumentConstraints
     {
         public static string StartsWith(this IArgumentConstraintManager<string> scope, string beginning)
         {
-            return scope.Matches(
-                x => x.StartsWith(beginning, StringComparison.CurrentCulture), string.Format(CultureInfo.CurrentCulture, "Starts with \"{0}\"", beginning));
+            return scope.Matches(x => x.StartsWith(beginning, StringComparison.CurrentCulture), $@"Starts with ""{beginning}""");

```

Comment:
I agree with separate PR. This, and #399, are only about formatting, which is a different purpose to that of comparison.